### PR TITLE
docs: update `INSTALL.md`

### PR DESCRIPTION
**Pull request by Mehrdad-Farshi**
### Description

Updates `INSTALL.md` use use `unzip` on Linux and macOS.

### CODE REVIEW

File: docs/INSTALL.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ The following example uses Bash (default) on macOS (Intel).
 3. Extract the plugin.
 
     ```console
-    tar xvf terraform-provider-vsphere_${RELEASE}_darwin_amd64.zip
+    gunzip terraform-provider-vsphere_${RELEASE}_darwin_amd64.zip
```
Comment (tenthirtyam) [inline docs/INSTALL.md:133]: Why not just `unzip` and provide the destination path?

```console
unzip terraform-provider-vsphere_${RELEASE}_darwin_amd64.zip -d terraform-provider-vsphere_${RELEASE}
```

Then

```console
mv terraform-provider-vsphere_v${RELEASE}/terraform-provider-vsphere_v${RELEASE} ~/.terraform.d/plugins/local/hashicorp/vsphere/${RELEASE}/darwin_amd64
```

File: docs/INSTALL.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ The following examples use Bash on Linux (x64).
 2. Extract the plugin.
 
     ```console
-    tar xvf terraform-provider-vsphere_${RELEASE}_linux_amd64.zip
+    gunzip terraform-provider-vsphere_${RELEASE}_linux_amd64.zip
```
Comment (tenthirtyam) [inline docs/INSTALL.md:85]: Could use just `unzip`.

File: docs/INSTALL.md (hunk 1 above)
Comment (Mehrdad-Farshi) [reply docs/INSTALL.md:133]: yes sure this is a much better approach

Comment (tenthirtyam) [reply docs/INSTALL.md:133]: Do you want to update your commit?

Comment (tenthirtyam) [reply docs/INSTALL.md]: Addressed in force push of 8e47f6f .

**Review by tenthirtyam — APPROVED** [review]
LGTM!

**Pull request merged**